src/components/add-form/index.js

```diff
@@ -18,38 +23,63 @@ const AddForm = ({ formState, handleForm }) => (
         required
       />
     </label>
-    <fieldset className="time-frame" defaultValue="7">
```

Comment:
This `defaultValue` was causing the bug? :eyes:

Reply:
In fact, with the new way we check whether the field is checked or not (`checked={formState.timeFrame === TimeFrames.soon}`), this `defaultValue` has become unnecessary, as we have defined the default behavior in `defaultValues` on `AddItemPage`.